sunkit_dem/util.py

```diff
@@ -0,0 +1,45 @@
+"""
+Utility functions
+"""
+import numpy as np
```

Comment:
```suggestion

__all__ = ["quantity_1d_to_sequence"]

import numpy as np
```

Reply:
I don't know why I didn't just hit the button and just commit these suggestions 😞